torch_nn_layers.py

```diff
@@ -91,12 +91,13 @@ class TorchAddLinearLayer(Component):
     out_features: InArg[int]
     bias: InArg[bool]
     model_out: OutArg[list]
-
+    
     def execute(self,ctx) -> None:
         bias = True if self.bias.value is None else False
         in_size = self.in_features.value
         out_size = self.out_features.value
-        
+        print("model_in value before adding Linear layer:", type(self.model_in.value))
```

Comment:
looks like a left-over debug print?

Reply:
yes, there was a bug here and  I forgot remove it

Reply:
Done